app/src/main/java/org/dslul/openboard/inputmethod/latin/userdictionary/UserDictionaryAddWordFragment.java

```diff
@@ -161,15 +184,18 @@ public void onPause() {
     public void onItemSelected(final AdapterView<?> parent, final View view, final int pos,
             final long id) {
         final LocaleRenderer locale = (LocaleRenderer)parent.getItemAtPosition(pos);
-        if (locale.isMoreLanguages()) {
+        // TODO: To be reactivated when UserDictionaryLocalePicker.UserDictionaryLocalePicker() is implemented
```

Comment:
What is `UserDictionaryLocalePicker`?

Reply:
When enabled, a new menu item appears with the title "More languages...".

But if you go to the UserDictionaryLocalePicker.java file, you'll see that the UserDictionaryLocalePicker() function is empty.
This would allow the user to choose the precise language in which he wishes to add his word. I don't see the point, as he can already do so by selecting "For all languages".

However, why not implement it in the future...

Reply:
But in your screenshot it looks like you can already choose the language

Reply:
> However, why not implement it in the future...

Usually the issue is that someone needs to actually do it. With all the ideas and feature requests I probably have work for a year...

Reply:
> What you can see in my screenshot is that you can choose between the "local" language system and "For all languages".
> You can't choose anything else.

Then the only thing missing is addition of more languages. Giving the choice of system languages + languages of enabled keyboard (includingsecondary languages) would be enough.

Reply:
You can get the locales using `getEnabledSubtypes` for the main locales, and get secondary locales for each using `Settings.getSecondaryLocales`

Reply:
I'm really sorry but I don't know how and where implement `getEnabledSubtypes` and `Settings.getSecondaryLocales`.
I'm lost with `local` written everywhere...

Reply:
You have some `getLocalesList` function, which seems to return existing user dictionary locales plus system locales. Here you could add the locales to the `locales` set.

Reply:
With this commit 7ad9d06, this should be good.

<details>
<summary><b>See screenshot</b></summary>
<br>
<img width=200 src=https://github.com/Helium314/openboard/assets/139015663/81674009-f872-4a0d-8ec7-90e1abbd2aa5>
</details>

Reply:
And with this commit 161e499, we get secondary locales in the language list.

Reply:
Looks great, thanks!